### PR TITLE
Update README.md, add CONTRIBUTING.md and sample configuration file

### PR DESCRIPTION
**Pull request by netniV**

### CODE REVIEW

**Comment by netniV** [comment]
Closing this as it's now part of a larger commit in #8

File: README.md [hunk 1]
```diff
@@ -77,39 +77,65 @@ X | ExoComp
...
+Both the DLL and the toml file must be placed in the STFC game folder which is always located
+at `C:\Games\Star Trek Fleet Command\Star Trek Fleet Command\default\game` as the following
+names:
 
-If you have Visual Studio Code, open the folder and it should ask you to install the CMake extensions bundle.  Once the extensions are installed, you can build the project by pressing `F7` or right clicking on the `CMakeLists.txt` file and selecting `Build All Projects`.  This may ask you to configure the project and pick various items to use in the build.  That should only occur once.
+- community_patch_settings.toml
+- version.dll
 
-If you do not have Visual Studio Code, this project uses CMake, so the simplest way to build it on Windows:
+The pre-compiled DLL can be downloaded from the official assets hosted at:
```
Comment (tashcan) [inline README.md:93]: As I mentioned in the other PR. I would probably just point to the Releases here, as that'll be where things will be for updates going forward.

File: README.md [hunk 2]
```diff
@@ -77,39 +77,65 @@ X | ExoComp
...
 
-First clone and initialise the repository:
+If you wish to contribute to the project, or simply compile the DLL yourself, please see [CONTRIBUTING.md](CONTRIBUTING.md)
 
-```bash
-git clone https://github.com/tashcan/bob.git
-cd bob
-git submodules update --recursive --init
-```
+## Installation / Runtme
```
[the remark this answers is not in the document]
Comment (netniV) [reply README.md:84]: Weird, that was fixed... in the other PR probably. I'll get that updated.

File: README.md (hunk 1 above)
Comment (netniV) [reply README.md:93]: I have been slow for the past day or two... I wondered where "here" was... thinking discord... until I just clicked... I'll have this updated shortly.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (netniV) [reply README.md:84]: Fixed.

**Comment by tashcan** [comment]
Thanks.

**Pull request merged**